DDD.Functions.Extensions/Repositories.cs

```diff
@@ -28,10 +27,11 @@ public static Task<(ITableStorageRepository<SessionEntity>, ITableStorageReposit
 
         public static async Task<(ITableStorageRepository<DedupeWebhookEntity>, IQueueStorageRepository<OrderNotificationEvent>, IQueueStorageRepository<TicketNotificationEvent>)> GetRepositoryAsync(this TitoWebhookConfig config)
         {
-            var storageAccount = CloudStorageAccount.Parse(config.ConnectionString);
-            var deDupeRepository = new TableStorageRepository<DedupeWebhookEntity>(storageAccount, config.DeDupeTable);
-            var orderNotificationQueue = new QueueStorageRepository<OrderNotificationEvent>(storageAccount, config.OrderNotificationQueue);
-            var ticketNotificationQueue = new QueueStorageRepository<TicketNotificationEvent>(storageAccount, config.TicketNotificationQueue);
+            var storageAccountForTables = Microsoft.Azure.Cosmos.Table.CloudStorageAccount.Parse(config.ConnectionString);
+            var storageAccountForQueues = Microsoft.Azure.Storage.CloudStorageAccount.Parse(config.ConnectionString);
```

Comment:
wow, that's weird

Reply:
(not your fault, the design of that microsoft api is weird)

Reply:
I was surprised to see these using different CloudAccount objects and AFAIK there is no way to unify them :(